frontend/pages/contact.js

```diff
@@ -66,43 +66,88 @@ const Contact = ({ joinUs, connectUs }) => {
+                                        />
+                                        <div className="hidden lg:block">Telegram</div>
+                                    </Button>
+                                </Link>
+                                <Link href="https://www.instagram.com/eduhopesg/" target="_blank" passHref>
+                                    <Button className={styles.contactSocialMedia} role="button">
+                                        <Image
+                                            className="rounded-full md:w-9 h-auto"
+                                            src="/images/instagram.png"
+                                            width={ICON_SIZE}
+                                            height={ICON_SIZE}
+                                            quality={ICON_QUALITY}
+                                            alt="Our Instagram account"
+                                        />
+                                        <div className="hidden lg:block">Instagram</div>
+                                    </Button>
+                                </Link>
+                                <Link href="https://www.tiktok.com/@eduhopesg" target="_blank" passHref>
+                                    <Button className={styles.contactSocialMedia} role="button">
+                                        <Image
+                                            className="rounded-full md:w-9 h-auto"
+                                            src="/images/tiktok.png"
+                                            width={ICON_SIZE}
+                                            height={ICON_SIZE}
+                                            quality={ICON_QUALITY}
+                                            alt="Our TikTok account"
+                                        />
+                                        <div className="hidden lg:block">TikTok</div>
+                                    </Button>
+                                </Link>
+                                <a href="mailto:eduhopesg@gmail.com">
+                                    <Button className={styles.contactSocialMedia} role="button">
+                                        <Image
+                                            className="rounded-full md:w-9 h-auto"
+                                            src="/images/email.png"
+                                            width={ICON_SIZE}
+                                            height={ICON_SIZE}
+                                            quality={ICON_QUALITY}
+                                            alt="Our email address"
+                                        />    
+                                        <div className="hidden lg:block">Email</div>
+                                    </Button>
+                                </a>
                             </div>
                         </div>
                     </div>
```

Comment:
This changes as well. Looks like you could have retained the "map" function to avoid unnecessary lines of code.

Reply:
Yep, i gets le. Will keep note of that.